Cargo.toml

```diff
@@ -19,7 +19,7 @@ prost-derive = "0.5"
 bytes = "0.4"
 futures = "0.1"
 grpcio = { version = "0.5.0-alpha.1", features = ["prost-codec"] }
-raft = { git = "https://github.com/pingcap/raft-rs.git", rev = "8e953b0d0c9cd2f98c94a42edc48ae3f09e967ff" }
+raft-proto = "0.1"
```

Comment:
Is this the right revision for TiKV? https://github.com/tikv/tikv/blob/master/Cargo.toml#L109

Reply:
TiKV is using a branch of KvProto at the moment since master is Prost-only, but this does break my Prost branch. I'll address it...